docs/sdk/src/reference_docs/omni_node.rs

```diff
@@ -177,9 +177,35 @@
 //! [This](https://github.com/paritytech/polkadot-sdk/issues/5565) future improvement to OmniNode
 //! aims to make such checks automatic.
 //!
+//! ### Runtime conventions
+//!
+//! Given OmniNode purpose is to be compatible with most Polkadot parachains, we thought it is
```

Comment:
nit: I'd prefer avoiding first-person plural. Could be reworked as:
Given OmniNode purpose is to be compatible with most Polkadot parachains, this chapter provides a  guidelines on how the developed parachain's runtimes should be composed.

Reply:
Agree! Rephrased this section anyway based on the request from @skunert: [0ba6ecb](https://github.com/paritytech/polkadot-sdk/pull/6450/commits/0ba6ecb244c1ae43bfc6262e2c1143bf3f735410)